packages/liquidator/src/liquidator.js

```diff
@@ -213,6 +213,20 @@ class Liquidator {
       at: "Liquidator",
       message: "Checking for liquidatable positions and preforming liquidations"
     });
+    if (
+      maxTokensToLiquidateWei &&
+      this.whaleDefenseFundWei &&
+      this.toBN(this.whaleDefenseFundWei).gte(this.toBN(maxTokensToLiquidateWei))
+    ) {
+      this.logger.info({
+        at: "Liquidator",
+        message:
+          "The whale defense fund reserve amount is greater than the liquidators synthetic balance. This might result in unintended consequences. Consider changing this param or adding more synthetics.",
```

Comment:
```suggestion
          "The whale defense fund reserve amount is greater than the liquidators synthetic balance. This might result in skipped liquidations. Consider reducing this threshold or adding more synthetics.",
```